tests/unit/amo/pages/TestGuides.js

```diff
@@ -276,6 +276,40 @@ describe(__filename, () => {
       );
     });
 
+    it.each([['does', 'not empty'], ['does not', 'empty']])(
```

Comment:
heh, that seems a bit convoluted and fragile but 🤷‍♂

Reply:
Yeah, this line is a bit confusing, but the output when the tests run is entirely understandable and it works well.